db/seeds.rb

```diff
@@ -86,6 +83,9 @@
     {name: 'MichaelS', actionkit_uri: '/rest/v1/tag/1160/'},
     {name: 'DeborahL', actionkit_uri: '/rest/v1/tag/1661/'},
     {name: 'KatieF', actionkit_uri: '/rest/v1/tag/1662/'}
+    {name: 'ChristianB', actionkit_uri: '/rest/v1/tag/1383/'}
```

Comment:
Oh, could you please add:

OliverD
MarieL

(two MVP writers)


On Fri, Mar 18, 2016 at 2:35 PM, Eric Boersma <notifications@github.com>
wrote:

> In db/seeds.rb
> <https://github.com/SumOfUs/Champaign/pull/435#discussion_r56655898>:
>
> > @@ -86,6 +83,9 @@
> >      {name: 'MichaelS', actionkit_uri: '/rest/v1/tag/1160/'},
> >      {name: 'DeborahL', actionkit_uri: '/rest/v1/tag/1661/'},
> >      {name: 'KatieF', actionkit_uri: '/rest/v1/tag/1662/'}
> > +    {name: 'ChristianB', actionkit_uri: '/rest/v1/tag/1383/'}
>
> Also, the line before the ones you added (KatieF)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SumOfUs/Champaign/pull/435/files/32071a717066e376711ca88a2122fafad1a999d9#r56655898>
>



-- 
*Eoin Dubsky*
Campaign Manager | SumOfUs.org
Phone: +31-6-41636410 (Netherlands)
Skype: eoin_dubsky